tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -436,7 +559,11 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
     "  def foo@@",
     // NOTE(olafur) I am not sure this is desirable behavior, we might want to
     // consider not dealiasing here.
-    """  def foo: List[Int] = ${0:???}""".stripMargin
+    """  def foo: List[Int] = ${0:???}""".stripMargin,
+    compat = Map(
+      "3" -> // TODO: should we dealias it?
```

Comment:
```suggestion
      "3" -> // TODO: should we dealias it?
```
I think it's actually correct not to dealias this.

Reply:
I think you can remove the TODO